lib/gems/pending/appliance_console/database_configuration.rb

```diff
@@ -272,5 +274,9 @@ def initialize_from_hash(hash)
         end
       end
     end
+
+    def default_port
```

Comment:
I think `attr_reader` implies something that can or might be changed. If I were to change this, I might actually make it a constant. Thoughts?